.travis.yml

```diff
@@ -1,46 +1,112 @@
-language: python3
-
 sudo: required
 
-addons:
-  firefox: 55.0
-  apt:
-    packages:
+matrix:
+  include:
+  - language: python3
+  
+    addons: 
+      firefox: 55.0
+  
+    apt:
+      packages:
       - python-dev
 
-python:
-  - 3.5
-cache:
-  directories:
-    - $HOME/.pip-cache/
+    python:
+      - 3.5
+    cache:
+      directories:
+        - $HOME/v1/.pip-cache/
```

Comment:
Why should we run on v1,the aim is to test front-end and backend. So pip cache must be present on the backend.

Reply:
@abishekvashok Exactly what I mentioned before..